package.json

```diff
@@ -33,6 +34,12 @@
     "@size-limit/preset-small-lib": "^4.10.1",
     "@solidity-parser/parser": "^0.12.0",
     "@types/object-hash": "^2.1.0",
+    "apollo": "^2.33.0",
+    "apollo-cache-inmemory": "^1.6.6",
+    "apollo-client": "^2.6.10",
+    "apollo-link-http": "^1.5.17",
+    "graphql": "^15.5.0",
+    "graphql-tag": "^2.12.4",
```

Comment:
These should be `dependencies`, not `devDependencies`

Reply:
oops, done